webroot/js/cake-core.js

```diff
@@ -0,0 +1,22 @@
+var elements = document.getElementsByClassName('cake-core-postLink');
```

Comment:
@TheFRedFox I meant that this script file should have a `<script ... ></script>` tag in the default layout so that it is included by default.